changelogs/current.yaml

```diff
@@ -128,6 +128,11 @@ bug_fixes:
     the number of requests per I/O cycle is configured and an HTTP decoder filter that pauses filter chain is present. This behavior
     can be reverted by setting the runtime guard ``envoy.reloadable_features.use_filter_manager_state_for_downstream_end_stream``
     to false.
+- area: http_proxy
```

Comment:
clarify this is with the proxy filter (which is arguably one folks can't use as there's no way to configure it outside of E-M)

Reply:
Done.